app/src/main/java/com/sereno/serenoandroidapp/MainActivity.kt

```diff
@@ -14,6 +15,7 @@ class MainActivity : AppCompatActivity() {
 
         val movePlayGifButton = findViewById<Button>(R.id.movePlayGifButton)
         val moveFirebaseTest = findViewById<Button>(R.id.moveFirebaseTestButton)
+        val moveWeather = findViewById<Button>(R.id.moveWeather)
```

Comment:
```suggestion
        val moveWeatherInfoButton = findViewById<Button>(R.id.moveWeatherInfoButton)
```

moveFirebaseTest間違ってるな(ごめん)